examples/window.rs

```diff
@@ -75,7 +87,10 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     tracing::init();
 
-    let event_loop = EventLoop::new()?;
+    let mut event_loop = EventLoop::new()?;
+    if let Some(event_loop) = event_loop.as_wayland_event_loop() {
+        event_loop.listen_linux_dmabuf::<App>();
+    }
```

Comment:
You can not really avoid that, but I guess it's fine. That's how at least what I remember from previous attempts with dynamically extending winit's functionality like that, so we don't bind globals, etc, that are no always needed and we don't plumb into `WindowEvent`, etc, etc.